packages/provider.mustache/spec/textInterpolationSpec.ts

```diff
@@ -251,7 +251,7 @@ describe('Interpolation Markup bindings', function () {
 
     jasmine.setNodeText(testNode, "hello {{'name'}}!");
     applyBindings(null, testNode);
-    expect(testNode).toContainText('hello --name--!');
+    expect(testNode).toContainText('hello --name--!'); */
```

Comment:
are we removing a test expect here?  do we lose some coverage?

Reply:
This test-case was already deactivited via `xit`, so we saved ourselves the type correction.